### ISSUE TITLE
Should we move Request to be team coordinator behind login?

### ISSUE THREAD

**Issue by tbock** [USEPA/GeoPlatform-Request-Tool#100]
I believe there was a reason for displaying team coordinators but does the request to be one need to be open?  Seems like requiring login for this request is not a big deal (they all have to have accounts already no?)

**Comment by tbock** [comment]
The changes in PR #77 moved the auth group list to a service call that is locked behind the login.  So it is currently blank if not logged in.

**Comment by tbock** [comment]
Sorry if I'm rehashing things.  But now that we have Resposne/Projects and those have Auth Groups... why does Team Coordinators need to even be associated with an auth group?

**Comment by jlherring** [comment]
So I never totally understood why the response request required login, but team coordinator did not. You aren’t a team coord yet, you wouldn’t have a log in for the request tool. 

If anything, whole coordinator page should have required login to see the list of existing names.

Is it too complicated to allow anyone to submit if they aren’t an existing coord (they have to provide all their info)? But if they are a coordinator, they have to login to pick their name from the list?

**Comment by tbock** [comment]
Well I'll add to the pile.  If they are requesting to be a coordinator they have to ALREADY(?) have an AGOL account.  Forcing them to login provides us with their name, email and username.  We could still offer those inputs but easily prepopulate from the AGOL user account. 

I think auth group got moved to response/projects at some point b/c we realized we couldn't always use the sponsors auth group.  A quick look tells me we don't use the sponsors auth group for anything anymore.  Only the response/project one.  Fairly certain its just left over from when we used to define the auth group at the sponsor level.  We can remove from the coordinator request form.
